vault/activity_log.go

```diff
@@ -1578,6 +1578,19 @@ type ResponseCounts struct {
 	NonEntityTokens  int `json:"non_entity_tokens" mapstructure:"non_entity_tokens"`
 	NonEntityClients int `json:"non_entity_clients" mapstructure:"non_entity_clients"`
 	Clients          int `json:"clients"`
+	SecretSyncs      int `json:"secret_syncs" mapstructure:"secret_syncs"`
+}
+
+func (existingRecord *ResponseCounts) Add(newRecord *ResponseCounts) {
```

Comment:
I've updated the receiver name to just be `r`. 

I do prefer having this be a method rather than a function, since (in the way it's currently implemented) it modifies only one of the records. I think if it was `func AddCountsToExisting(existing, new *ResponseCounts)` it would be very easy for the caller to not realize which struct got modified as a result of this function. I try to follow this: https://dave.cheney.net/2019/09/24/be-wary-of-functions-which-take-several-parameters-of-the-same-type.

Alternatively, we could do `func AddCountsToExisting(existing, new *ResponseCounts) *ResponseCounts` so that neither `existing` or `new` is modified, but then we're just creating intermediary structs for no reason.

Reply:
Sounds good to me!